src/components/forms/validator.ts

```diff
@@ -68,7 +68,7 @@ export const mustBeEthereumAddress = memoize(
     const startsWith0x = address?.startsWith('0x')
     const isAddress = getWeb3().utils.isAddress(address)
 
-    const errorMessage = `Input must be a valid Ethereum address${
+    const errorMessage = `Must be a valid checksummed address${
```

Comment:
It doesn't have to be checksummed, does it? It would still accept all lowercase addresses.

Reply:
I'm not sure @tschubotz @rmeissner ?

Reply:
Nope, not a requirement that it's checksummed. We should allow to input all lowercase addresses as well!

Reply:
fixed